app/utils/get_filtered_records.py

```diff
@@ -144,33 +144,22 @@ def get_filtered_records(research_fields=False, filters=None, columns=None,
     if query_dicts is None or len(query_dicts) == 0:
```

Comment:
I think we can be sure query_dicts will never be None, so we can remove that from this conditional